src/devices_models/devices/common/constraints_structs.jl

```diff
@@ -1,12 +1,19 @@
 """ Data Container to construct range constraints"""
 struct DeviceRange
+    name::String
     limits::MinMax
     additional_terms_ub::Vector{Symbol}
     additional_terms_lb::Vector{Symbol}
 end
 
+function DeviceRange(name::String, limits::MinMax)
+    return DeviceRange(name, limits, Vector{Symbol}(), Vector{Symbol}())
+end
+
 struct DeviceTimeSeries
+    name::String
     bus_number::Int64
     multiplier::Float64
     timeseries::Vector{Float64}
+    range::Union{Nothing,DeviceRange}
```

Comment:
This is where the name would be stored twice

Reply:
If DeviceRange will be used independently of DeviceTimeSeries then I don't have a problem with name being stored twice.  I would suggest putting an `assert name == range.name` in the DeviceTimeSeries constructor.

Reply:
ok. 

P.S. @claytonpbarrows you are missing a space after comma here too :P

Reply:
I added the assertion statement in an inner constructor. @daniel-thom is that the best way to do it?
And to answer your question directly... yes, `DeviceRange` is used independently of `DeviceTimeSeries`

Reply:
@daniel-thom I kept the assert inside to keep the logic correct... if the range isn't nothing, make sure that the names match...